chainstate/client.go

```diff
@@ -166,6 +173,11 @@ func (c *Client) BroadcastClient() broadcast.Client {
 	return c.options.config.broadcastClient
 }
 
+// PulseClient will return the Pulse client
+func (c *Client) PulseClient() *PulseClient {
```

Comment:
Do we ned to actually expose that one? Are we using this method?

Reply:
Removed